src/com/textFinder/TextFinderImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.textFinder;
```

Comment:
Create two scripts:

`./build.sh` and `./run.sh --... same flags`

`./build.sh`: Will build the java application into jar

`./run.sh --...` Will run the jar and forward the arguments to the java app